cmake/modules/FindMySqlClient.cmake

```diff
@@ -21,7 +21,7 @@ else()
   set(EXTRA_FIND_ARGS)
 endif()
 
-find_path(MYSQLCLIENT_INCLUDE_DIR mysql/mysql_time.h)
+find_path(MYSQLCLIENT_INCLUDE_DIR NAMES mysql/server/mysql.h mysql/mysql.h)
```

Comment:
please change the order here. We want to find mysql first.

Reply:
sure - I've updated the PR.